easybuild/easyconfigs/g/GeneMark-ET/GeneMark-ET-4.65-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'Tarball'
+
+name = 'GeneMark-ET'
+version = '4.65'
+
+homepage = 'http://exon.gatech.edu/GeneMark'
+description = "Eukaryotic gene prediction suite with automatic training"
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+# download via http://exon.gatech.edu/GeneMark/license_download.cgi
+sources = ['gmes_linux_64.tar.gz']
```

Comment:
```suggestion
# rename gmes_linux_64.tar.gz to gmes_linux_64-4.65.tar.gz
sources = ['gmes_linux_64-%(version)s.tar.gz']
```

We should keep the version info in the download instructions and sources.

Reply:
Thank you @branfosj, that does make sense.  I've made this change.